frontends/web/src/app.tsx

```diff
@@ -149,13 +147,13 @@ class App extends Component<Props, State> {
       route('/', true);
       return;
     }
-    // if on an account that isnt registered route to /
+    // if on an account that isn't registered route to /
     if (inAccounts && !accounts.some(account => currentURL.startsWith('/account/' + account.code))) {
       route('/', true);
```

Comment:
@thisconnect this is executed first if you are in an account which is removed when you unplug.